pdns/dnsdistdist/dnsdist-lua-bindings-packetcache.cc

```diff
@@ -90,6 +93,11 @@ void setupLuaBindingsPacketCache(LuaContext& luaCtx, bool client)
         if (vars->count("cookieHashing")) {
           cookieHashing = boost::get<bool>((*vars)["cookieHashing"]);
```

Comment:
I wonder if it wouldn't be cleaner to insert into `optionsToSkip` and get rid of `DNSDistPacketCache::setCookieHashing` instead? It does not seem to be used anywhere else, except in `fuzz_dnsdistcache.cc` where we will need to use the new options anyway.